src/lib/clk.c

```diff
@@ -63,6 +70,9 @@ void clock_set_freq(int clock, uint32_t hz)
 					 hz);
 	clk_notify_data.freq = clk_info->freqs[idx].freq;
 
+	tr_info(&clock_tr, "clock %d set freq %dkHz freq_idx %d",
+		clock, hz / 1000, idx);
```

Comment:
xtensa divison is slow. Lets send Hz and have logger do any conversions (if needed). i.e. we send all our trace frequencies in Hz and have logger do all conversions. The logger conversion can come later though in another PR.

Reply:
updated - conversion was done only to limit number of `0` in output log, I think that making hack's in logger for such a thing is not needed, it's not important enough.